aws-lc-rs/src/aead.rs

```diff
@@ -49,8 +49,8 @@
 //!
 //! # TLS AEAD APIs
 //!
-//! Application developers developing TLS protocol implementations, with FIPS compliance requirements
-//! should use the [`TlsRecordSealingKey`] and [`TlsRecordOpeningKey`] respectively.
+//! Application developers developing TLS protocol implementations should use
```

Comment:
NP: “Application developer” seems like the wrong term for someone who's implementing the TLS protocol. Maybe “Systems developer”?